pkg/v1/endpoints/books/request.go

```diff
@@ -0,0 +1,14 @@
+package books
+
+import (
+    "github.com/King-Rullmann-Book-Club/books-api/pkg/v1/service/books"
+)
+
+type GetBookRequest struct {
+    ID string
+}
+
+type GetBookResponse struct {
+    Book books.Book `json:"book,omitempty"`
```

Comment:
Not sure if this is intentional, but having this be a go object means that the response has non-standard json casing.

```
$ curl localhost:8080/books/1
{"book":{"Name":"Perdido Street Station"}}
```
Note the capitalization of `Name`.

Could this be solved by changing the interface of Book like so?
```
type Book struct {
    Name string `json:"name"`
}
```

Reply:
Yes it can. I'll update.